drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -688,7 +690,6 @@ local function do_bridge_network_init(driver, device, bridge_url, api_key)
   if not device:get_field(Fields.EVENT_SOURCE) then
     log.info_with({ hub_logs = true }, "Creating SSE EventSource for bridge " ..
       (device.label or device.device_network_id or device.id or "unknown bridge"))
-    device:offline()
```

Comment:
Will the bridge ever be marked offline if we can't establish a connection?

Reply:
Yes, if the connection was connected and then errored out, or there was an error when connecting. I will manually test this case again though to be sure.

Reply:
I tested this by disconnecting the bridge and restarting the driver. As the driver restarted, the bridge (and children) were correctly marked as offline.

Reply:
Now using the correct test setup - I still that the hue bridge is correctly marked offline at startup when it is disconnected.